addon/components/loading-button.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+
+const {
+  Component,
+  RSVP
+} = Ember;
+
+const LoadingButtonComponent = Component.extend({
+  tagName: 'button',
+  classNameBindings: ['isLoading:js-btn--loading'],
+  attributeBindings: ['isLoading:disabled'],
+
+  click(e) {
+    e.preventDefault();
+
+    let originalContent = this.$().html();
+    let originalWidth = this.$().width();
+
+    // Because `.send` method for sending actions does not return anything, we
+    // pass it an `RSVP.defer` to be resolved in the remote action.
+    let defer = RSVP.defer();
+    defer.promise.then(() => {
+      this.$().removeAttr('style');
```

Comment:
Because when i set the `loading button width` to the original button width, it's set a `style="width: ..."`.